### ISSUE TITLE
Regression: numeric settings are now read as character arrays

### ISSUE THREAD

**Issue by raacampbell** [alexanderbrown/masiv#15]
It looks like the changes for reading sample IDs starting with a number or a zero have screwed up the reading of the settings file. So it now reads numeric settings as strings which made me think I had a bug here: 83b1568f5ebe7c7dce6db979513f2fd05f6feea8 That fixed it by accident but now we can't pan because the pause setting for the keyboard pan is broken.

**Comment by raacampbell** [comment]
Hmmm... So it's just crap that starts with a zero. 
```
>> m=masiv.yaml.readSimpleYAML('masivPrefs.yml');m.navigation

ans = 

  struct with fields:

            panIncrement: [10 120]
         scrollIncrement: [10 1]
                zoomRate: 1.5000
           panModeInvert: '0'
        scrollZoomInvert: 1
       scrollLayerInvert: 1
    keyboardUpdatePeriod: '0.005'
```

**Comment by raacampbell** [comment]
I'll do this. I'll scale the heights of neat software development by doing it with a unit test.

**Comment by alexanderbrown** [comment]
Argh I knew what I was doing was hacky.

**Comment by raacampbell** [comment]
I'm on it right now.

**Comment by alexanderbrown** [comment]
Right. This will of course have been caused by the fix for the sample ID.
Anything numeric with a leading zero is kept as a string by readSimpleYAML.
Perhaps a fix is to change this to anything numeric, which is not in fact
equal to zero?

**Comment by raacampbell** [comment]
Yes, that seems reasonable.

**Comment by raacampbell** [comment]
Except that this won't catch the problematic YAML that we were trying to fix, since there `stackName` was equal to `01` with no characters. I think the only way to fix that would be to force it to read certain settings as a settings (or certain files) a string.

**Comment by alexanderbrown** [comment]
Or to force the user to escape the stackname e.g. '01' in the file, and
detect this

**Comment by raacampbell** [comment]
The saga continues: fd8fb9e. I think we're now good. It passes the tests and MaSIV works again.

**Comment by alexanderbrown** [comment]
Looks good, makes sense. 

Logically, there _should_ be no number, saved by matlab, that is a whole number, with leading zeros. 

He says, fingers crossed.
